java/alb-multi-rule-response/README.md

```diff
@@ -0,0 +1,44 @@
+# ALB using a combination of host and path based routing with Java AWS-CDK 
+### The ALB is backed by an EC2 instance acting as frontend client and built-in ALB response rules that simulates mobile and application api
+
+## Testing the responses
+### testing mobile produciton api
+``` bash 
+curl -H "Host: mobile.mydomain.com" [ALB-DNS-FQDN]/production
+```
+Resonse received
+```json
+{"status":"succes","apiversion":"prod_mobile_v1"}
+```
+
+### Testing frontend production api
+``` bash 
+curl -H "Host: api.mydomain.com" [ALB-DNS-FQDN]/production
+```
+
+```json
+{"status":"succes","apiversion":"prod_api_v1"}
+```
+
+## List of ALB rules
+![alt text](imgs/screenshot.png "screenshot of resulting rule")
+
+
+It is a Maven-based project, so you can open this directory with any Maven-compatible Java IDE, and you should be able to build and run tests from your IDE.
+
+You should explore the contents of this template. It demonstrates a CDK app to create a codebuild Project. 
```

Comment:
Not sure this is a Codebuild project! 😉

Put a description in brief of the functionality this demonstrates.